tracing/src/main/java/org/axonframework/extensions/tracing/TraceableSubscriptionQueryResult.java

```diff
@@ -0,0 +1,55 @@
+package org.axonframework.extensions.tracing;
+
+import io.opentracing.Span;
+import org.axonframework.common.Registration;
+import org.axonframework.queryhandling.SubscriptionQueryResult;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * Traceable implementation of {@link SubscriptionQueryResult}.
+ *
+ * @param <I> The type of initial result
+ * @param <U> The type of incremental updates
+ * @author Lucas Campos
+ * @since 4.2
+ */
+public class TraceableSubscriptionQueryResult<I, U> implements SubscriptionQueryResult<I, U> {
```

Comment:
Might be nice to have a dedicated unit test class for this guy, even though it's gonna be pretty simple.

Reply:
Sure, added a unit test for it.